notebooks/MAST/HSC/CASJOBS/HSCV3_API/hscv3_api.ipynb

```diff
@@ -0,0 +1,636 @@
+    "    if release not in releaselist:\n",
+    "        raise ValueError(f\"Bad value for release (must be one of {', '.join(releaselist)})\")\n",
+    "    if release == \"v2\":\n",
+    "        tablelist = (\"summary\", \"detailed\")\n",
+    "    else:\n",
+    "        tablelist = (\"summary\", \"detailed\", \"propermotions\", \"sourcepositions\")\n",
+    "    if table not in tablelist:\n",
+    "        raise ValueError(f\"Bad value for table (for {release} must be one of {', '.join(tablelist)})\")\n",
+    "    if table == \"summary\":\n",
+    "        magtypelist = (\"magaper2\", \"magauto\")\n",
+    "        if magtype not in magtypelist:\n",
+    "            raise ValueError(f\"Bad value for magtype (must be one of {', '.join(magtypelist)})\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Get metadata on available columns <a name=\"metadata\"></a>\n",
+    "\n",
+    "The `metadata` query returns information on the columns in the table.  It works for any of the tables in the API (`summary`, `detailed`, `propermotions`, `sourcepositions`).\n",
+    "\n",
+    "Note that the summary table has a huge number of columns!  Each of the 133 filter/detector combinations has 3 columns with the magnitude, median absolute deviation (MAD, a robust measure of the scatter among the measurements), and the number of independent measurements in the filter.  The filter name includes a prefix for the detector (`A`=ACS/WFC, `W3`=WFC3/UVIS or WFC3/IR, `W2`=WFPC2) followed by the standard name of the filter.  So for instance all three instruments have an F814W filter, so there are columns for `A_F814W`, `W3_F814W`, and `W2_F814W`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "meta = hscmetadata(\"summary\")\n",
+    "print(len(meta), \"columns in summary\")\n",
+    "filterlist = meta['name'][19::3].tolist()\n",
+    "print(len(filterlist), \"filters\")\n",
+    "pprint(filterlist, compact=True)\n",
+    "meta[:19]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Find variable objects in the dwarf irregular galaxy IC 1613 <a name=\"ic1613\"></a>\n",
+    "\n",
+    "This is based on [HSC Use Case #3](https://archive.stsci.edu/hst/hsc/help/use_case_3_v2.html), which shows an example of selecting objects from the HSC in portal.  This is simple to do using the HSC API."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Use `astropy` name resolver to get position of IC 1613 <a name=\"resolver\"></a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "target = 'IC 1613'\n",
+    "coord = SkyCoord.from_name(target)\n",
+    "\n",
+    "ra = coord.ra.degree\n",
+    "dec = coord.dec.degree\n",
+    "print(f'ra: {ra}\\ndec: {dec}')"
```

Comment:
```suggestion
    "coord\n",
```
I would suggest instead doing this, and then in the places below where `ra` and `dec` are used, do `coord.ra.deg` and `coord.dec.deg`.  That's clearer because `ra` and `dec` on their own are reused as variable names in several places, and its easy to get those "out of sync" with the `coord` - the `SkyCoord` objects were meant specifically to prevent that by carrying all the info in one variable.

Reply:
Tagging @rlwastro 

See this [comment](https://github.com/spacetelescope/notebooks/pull/203#discussion_r937968360). As stated, I think multiple uses of `ic1613_coord.dec.deg` is verbose compared to `dec_ic1613`.

I think a bigger issue is using the same variable name for inner and out scope.